### PR TITLE
regenerate hammer, hammer-foreman, hammer-katello

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec [hunk 1]
```diff
@@ -1,64 +1,60 @@
...
 %{!?_root_bindir:%global _root_bindir %{_bindir}}
 %{!?_root_mandir:%global _root_mandir %{_mandir}}
 %{!?_root_sysconfdir:%global _root_sysconfdir %{_sysconfdir}}
+%global hammer_confdir %{_root_sysconfdir}/hammer
 
-Summary: Universal command-line interface for Foreman
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 2.2.0
 Release: %{?prerelease:0.}%{release}%{?prerelease}%{?nightly}%{?dist}
+Summary: Universal command-line interface
```
Comment (ekohl) [inline packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec:19]: the `for Foreman` part was nice

File: packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec [hunk 2]
```diff
@@ -102,33 +97,36 @@ mkdir -p %{buildroot}%{_root_mandir}/man1
...
+%license %{gem_instdir}/LICENSE
 %{gem_instdir}/bin
-%{gem_instdir}/lib
+%{gem_libdir}
 %{gem_instdir}/locale
-%license %{gem_instdir}/LICENSE
 %exclude %{gem_cache}
 %{gem_spec}
+%{hammer_confdir}/cli.modules.d
+%config %{hammer_confdir}/cli_config.yml
```
Comment (ekohl) [inline packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec:117]: Looks like this is true for all, but does the `(noreplace)` part make sense to keep and include in the template? Are we consistent with all hammer (plugins) now?

Comment (evgeni) [reply packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec:117]: When this is merged, we're consistent with all plugins not having "noreplace"

File: packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec (hunk 1 above)
Comment (evgeni) [reply packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec:19]: *technically* `hammer_cli` is not for Foreman, `hammer_cli_foreman` is :)

I don't mind changing it back, but we should also maybe update the gem descriptions then?

**Pull request merged**